sensor_msgs/Image frame
---
string info
Target[] objects
